package.json

```diff
@@ -57,6 +58,11 @@
         "category": "Python"
       },
       {
+        "command": "python.setInterpreter",
+        "title": "Set Workspace Interpreter",
```

Comment:
Please change the title to "Select Workspace Interpreter", as that's all you can do - select it.
When the VS Code api becomes available, we can change this to "Set Workspace Interpreter"